ietf/context_processors.py

```diff
@@ -28,11 +28,11 @@ def secondary_menu(site):
 def social_menu(site):
     social = SocialMediaSettings.for_site(site)
     links = [
-        {"url": social.linkedin, "icon": "linkedin"},
-        {"url": social.twitter, "icon": "twitter"},
-        {"url": social.youtube, "icon": "youtube"},
-        {"url": social.mastodon, "icon": "mastodon"},
-        {"url": social.github, "icon": "github"},
+        {"url": social.linkedin, "icon": "linkedin", "title": "LinkedIn"},
+        {"url": social.twitter, "icon": "twitter", "title": "Twitter"},
```

Comment:
Not specific to this PR, but the name and icon probably need to be updated at some point.

Reply:
Might be better to create an issue so we don't forget about name change.

Reply:
Happy to do that

Reply:
Done: https://github.com/ietf-tools/www/issues/459